plaso/parsers/jsonl_plugins/gcp_log.py

```diff
@@ -33,27 +49,46 @@ class GCPLogEventData(events.EventData):
     request_target_tags (str): TODO
     resource_labels (list[str]): resource labels.
     resource_name (str): name of the resource.
+    service_account_delegation (list[str]): service accounts delegation in the
+        authentication.
     service_account_display_name (str): display name of the service account.
-    service_name (str): name of the servie.
+    service_account_key_name (str): service account key name used in
+        authentication.
+    service_name (str): name of the service.
     severity (str): log entry severity.
+    source_images (list[str]): source images of disks attached to a compute
+        engine instance.
+    status_code (str): operation success or failure code.
+    status_message (str); operation success or failure message.
     text_payload (str): text payload for logs not using a JSON or proto payload.
     user (str): user principal performing the logged action.
+    user_agent (str): user agent used in the request.
   """
 
   DATA_TYPE = 'gcp:log:entry'
 
   def __init__(self):
     """Initializes event data."""
     super(GCPLogEventData, self).__init__(data_type=self.DATA_TYPE)
+    self.caller_ip = None
     self.container = None
+    self.dcsa_email = None
```

Comment:
since this is a list, perhaps it could be named dcsa_emails instead?

Reply:
Fixed.